filehandler.cpp

```diff
@@ -58,3 +62,52 @@ QStringList FileHandler::getFileContent(QString filePath) {
 
   return list;
 }
+
+void FileHandler::saveHighlightList(QList<HighlightData> highList) {
+  QFile file(highlightFileName);
+  file.open(QIODevice::WriteOnly | QIODevice::Text);
+  file.resize(0);
+
+  QTextStream stream(&file);
+
+  for (int i = 0; i < highList.size(); i++) {
```

Comment:
Aqui é um oportunidade boa de usar aquele range based iterator do c++11.

`for (auto highlight : highList) {
     QString line = highlight.text() + "," + highlight.foreColor().name() + "," +
                    highlight.backColor().name() + "\n";

      qDebug() << line;

      stream << line;
   }`

Reply:
Podes crer. Codigo de dezembro ta todo depreciado tecnicamente já. haha!